recipes/prince/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "prince" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f6dc02fa63bf2b1b0a5edb7c1f30efc062e498bbf705a207b8bb5e1fa3fec4fa
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
```

I think this recipe can be ``noarch`` and you can restrict the ``python`` version in the ``requirements``
```suggestion
  noarch: python
```